packages/react/src/components/AspectRatio/AspectRatio.js

```diff
@@ -62,7 +62,16 @@ AspectRatio.propTypes = {
    * Specify the ratio to be used by the aspect ratio container. This will
    * determine what aspect ratio your content will be displayed in.
    */
-  ratio: PropTypes.oneOf(['16x9', '9x16', '2x1', '1x2', '4x3', '3x4', '1x1']),
+  ratio: PropTypes.oneOf([
+    '16x9',
+    '9x16',
+    '2x1',
+    '1x2',
+    '4x3',
+    '3x4',
+    '3x2',
```

Comment:
```suggestion
    '3x2',
    '2x3',
```

Would be awesome to include `2x3` in there too if you have the chance!

Reply:
Done!

Reply:
🙌